src/web/layouts/ShowcaseLayout/ShowcaseLayout.tsx

```diff
@@ -31,9 +30,9 @@ import { parse } from '@frontend/lib/slot-machine-flags';
 
 import GE2019 from '@frontend/static/badges/general-election-2019.svg';
 
-import { decideLineCount, decideLineEffect } from './layoutHelpers';
-import { Border } from './Border';
-import { GridItem } from './GridItem';
+import { decideLineCount, decideLineEffect } from '../layoutHelpers';
+import { Border } from '../Border';
```

Comment:
Do you think it would be useful to lint absolute paths from root instead of relative? Might be better DX (general question really, @oliverlloyd and I have discussed briefly before)

Reply:
I agree, we should use absolute unless the file is located in the same folder